manifests/init.pp

```diff
@@ -61,6 +61,9 @@
   $use_tls                  = false,
   $ca_file                  = undef,
   $permitted_peer           = undef,
+  $umask                    = '0000',
+  $file_create_mode         = '0644',
+  $dir_create_mode          = '0750',
```

Comment:
default should be '0700' as per the documentation

http://www.rsyslog.com/doc/rsconf1_dircreatemode.html